server/command.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+const COMMAND_HELP = `* |/zoom start| - Start a zoom meeting.`
+
+func getCommand() *model.Command {
+	return &model.Command{
+		Trigger:          "zoom",
+		DisplayName:      "Zoom",
+		Description:      "Integration with Zoom.",
+		AutoComplete:     true,
+		AutoCompleteDesc: "Available commands: start",
+		AutoCompleteHint: "[command]",
+	}
+}
+
+func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: args.ChannelId,
+		Message:   text,
+	}
+	_ = p.API.SendEphemeralPost(args.UserId, post)
+}
+
+func (p *Plugin) executeCommand(c *plugin.Context, args *model.CommandArgs) (string, error) {
+	config := p.getConfiguration()
+
+	split := strings.Fields(args.Command)
+	command := split[0]
+	action := ""
+
+	if command != "/zoom" {
+		return &model.CommandResponse{}, nil
+	}
+
+	if len(split) > 1 {
+		action = split[1]
+	} else {
+		p.postCommandResponse(args, "Please specify an action for /zoom command.")
+		return &model.CommandResponse{}, nil
```

Comment:
You also have to change the return values. E.g. the two lines above should be:
```suggestion
		return "Please specify an action for /zoom command.", nil
```